### PR TITLE
Issue-398:Added functionality to decode MAP+MessageFrame using .uper file extension

### PR DESCRIPTION
**Pull request by paulbourelly999**
# PR Details
## Description

Added functionality to MapPlugin to process files with extension '.uper' which will expect hex encoded MAP messages including the message frame. This is a work around for some issues encountered with processing other formats of MAP messages.
## Related Issue
#398 

## Motivation and Context
Work around for issues encountered attempting to processes MAP files without  message frame included.

## How Has This Been Tested?
Tested locally 

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [ x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] I have read the **CONTRIBUTING** document.
[V2XHUB Contributing Guide](https://github.com/usdot-fhwa-OPS/V2X-Hub/blob/develop/Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/v2i-hub/CARMAStreetsPlugin/manifest.json [hunk 1]
```diff
@@ -19,6 +19,11 @@
         "type": "J2735",
         "subtype": "BSM",
         "description": "Basic Safety Message"
+    },
+    {
+        "type":"J2735",
```
Comment (paulbourelly999) [inline src/v2i-hub/CARMAStreetsPlugin/manifest.json:24]: Add MAP message to manifest json for CARMA-Streets to allow tmx message counts.

File: src/v2i-hub/MapPlugin/manifest.json [hunk 2]
```diff
@@ -18,9 +18,14 @@
...
 		},
+		{
+			"key":"LogLevel",
+			"default":"INFO",
+			"description": "Map Plugin Log Level controls which log statements are printed to the terminal."
+		},
 		{
 			"key":"MAP_Files",
-			"default":"{ \"MapFiles\": [\n  {\"Action\":0, \"FilePath\":\"GID_Telegraph-Twelve_Mile_withEgress.xml\"}\n] }",
+			"default":"{ \"MapFiles\": [ {\"Action\":0, \"FilePath\":\"/var/www/plugins/MAP/MAP_9709_UPER.txt\"}] }",
```
Comment (paulbourelly999) [inline src/v2i-hub/MapPlugin/manifest.json:28]: Remove end line which causes issues for updating configuration parameters

File: src/v2i-hub/MapPlugin/manifest.json [hunk 3]
```diff
@@ -18,9 +18,14 @@
 			"default":"1000",
 			"description":"The frequency to send the MAP message in milliseconds."
 		},
+		{
+			"key":"LogLevel",
```
Comment (paulbourelly999) [inline src/v2i-hub/MapPlugin/manifest.json:22]: Add plugin loglevel configuration parameter

**Comment by paulbourelly999** [comment]
@jwillmartin Let me know if you have any concerns, also once you are OK with the changes feel free to approve the PR and squash and merge it. Thanks again.

**Review by jwillmartin — COMMENTED** [review]
All looks good to me

**Pull request merged**